### PR TITLE
Add subject, general and imaging metadata

### PR DESCRIPTION
**Pull request by alessandratrapani**

### CODE REVIEW

**Comment by h-mayorquin** [comment]
Todo:
- [ ]  Add keywords.
- [ ] Add genetict line.
- [ ] Add subject description.
- [ ] Add manufacturer for the microscope device.
- [ ] Add surgery
- [ ] Add channel information [see error]

**Comment by alessandratrapani** [comment]
> * [ ]  Add channel information [see error]
#13

**Comment by h-mayorquin** [comment]
@alessandratrapani This is still not working. It is failing for the keywords. Can you fix it, check the metadata one more time and then ping me to review it again?

File: src/clandinin_lab_to_nwb/brezovec/brezovec_metadata.yaml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 NWBFile:
+  keywords: [keyword1, keyword2]
```
Comment (h-mayorquin) [inline src/clandinin_lab_to_nwb/brezovec/brezovec_metadata.yaml:2]: We should have filled these.

**Pull request merged**